root/genre/DeleteGenre.js

```diff
@@ -9,33 +9,39 @@
 
 import * as React from 'react';
 
-import FormSubmit from '../components/FormSubmit';
+import EnterEdit from '../components/EnterEdit';
+import EnterEditNote from '../components/EnterEditNote';
 import EntityLink from '../static/scripts/common/components/EntityLink';
 
 import GenreLayout from './GenreLayout';
+import type {GenreDeleteFormT} from './types';
 
 type Props = {
   +$c: CatalystContextT,
-  +genre: GenreT,
+  +entity: GenreT,
+  +form: GenreDeleteFormT,
 };
 
 const DeleteGenre = ({
   $c,
-  genre,
+  entity,
```

Comment:
You can supply an alias when destructuring to keep the previous variable name, too.

```suggestion
  entity: genre,
```